### ISSUE TITLE
NO_PROXY support

### ISSUE THREAD

**Issue by joel23888** [JFrogDev/jfrog-cli-go#140]
Does jfrog support the NO_PROXY environment variable? I have to use a proxy to use the internet but my Artifactory server requires bypassing the proxy. The only way I can use jfrog at the moment is to clear and then restore the http_proxy environment variable when using jfrog. It would be convenient if it could support NO_PROXY.

Environment: jfrog version 1.14.0 running Windows 7.

**Comment by yahavi** [comment]
@joel23888,
Yes, JFrog CLI does support the `no_proxy` environment variable. 

In order to bypass proxy for Artifactory, you can set `no_proxy` environment variable with Artifactory's IP.

Please let us know if you have any issues with it.

**Comment by eyalbe4** [comment]
@joel23888,
Did you get a chance to try and follow yahavi's suggestion?

**Comment by joel23888** [comment]
I followed the suggestion but it does not work. Let’s say my artifact is at ‘test-repo/test’.

I first set NO_PROXY environment variable in Windows to the IP of my Artifactory server. 

I then ran cmd and ran jfrog config. I entered my server details and API key. These work from another system which does not require proxy bypass so I know they are correct. Also the server details work if I just use curl from the same (proxy bypass required) PC.

I then ran ‘jfrog rt dl test-repo/test’ and it just says it is looking for artifacts.
